eng/Publishing.props

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <PublishingVersion>3</PublishingVersion>
+    <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
+    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishPackages</PublishDependsOnTargets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_PackagesToPublish Remove="@(_PackagesToPublish)" />
+    <_PackagesToPublish Include="$(ArtifactsDir)nupkgs/*.nupkg" UploadPathSegment="nuget-client" Condition="'$(DotNetBuildFromSource)' == 'true'" />
+    <_SymbolsPackages Include="$(ArtifactsDir)nupkgs/*.symbols.nupkg" />
+    <_PackagesToPublish Remove="@(_SymbolsPackages)" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+  </ItemGroup>
+
+  <Target Name="_PublishPackages">
```

Comment:
This looks odd. nuget client has no PostBuildSign support and and no build outside of Windows_NT. It looks like this was maybe copied from aspnetcore.

Also, why the extra target? Shouldn't it be sufficient to simply have arcade import Publishing.props, then fill up ItemsToPushToBlobFeed from _PackagesToPublish?

Reply:
Arcade will not collect any packages, as they are not in `artifacts/Packages` dir, but in `artifacts/nupkgs`. https://github.com/dotnet/arcade/blob/e567828289357d7ca4d0c21e422a45bf5ce798a7/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L73-L74